index.html

```diff
@@ -48,17 +54,17 @@ <h2 class="tx-center">Full-Stack Software Developer</h2>
   <!--Documentation section-->
   <section class="container container-wrap" id="documentation">
```

Comment:
- [ ] This section is looking good, but there is an issue with your elements inside. If you check the original Figma template, you will realize that `Frameworks`, `Languages`, and `Skills` are all centered and in the same line, but for your design, you have only `Languages`, and `Frameworks` in the same line and then you have `Skills` in a new one. Even though this does not break your site because it is still responsive, please make sure to center these elements in the same line to stick to the original design as much as possible.

Reply:
Figma template
![image](https://user-images.githubusercontent.com/98621809/154823158-cd954919-56c9-434c-89db-ba04aa6f8b84.png)

Mobile version from inspector
![image](https://user-images.githubusercontent.com/98621809/154823176-7fd9a005-d687-4160-9fab-a35014d614a3.png)

Figma template desktop version
![image](https://user-images.githubusercontent.com/98621809/154823190-c6daec85-1d2c-4b79-a8ea-c68d659384b9.png)

Desktop version from inspector
![image](https://user-images.githubusercontent.com/98621809/154823206-99d1a093-5a33-4546-a662-22646cd37349.png)